uro98/assignment2/BinaryTree.java

```diff
@@ -0,0 +1,20 @@
+/**
+ * Represents a generic binary tree.
+ *
+ */
+public class BinaryTree<T> {
+  
+  Node<T> root;
+  
+  /**
+   * Set the root node to the value specified.
+   * @param data a generic value
+   */
+  public void addRoot(T data) {
```

Comment:
Optional: for convenience of use this method could return newly created `Node` so that users can easily use it to continue building the tree.